build.sbt

```diff
@@ -1,82 +1,45 @@
-import org.scalajs.linker.interface.ModuleSplitStyle
-import sbtcrossproject.CrossPlugin.autoImport.{ CrossType, crossProject }
-import sbt._
-import sbt.io.Using
+val scala213 = "2.13.10"
+val scala3   = "3.2.1"
+ThisBuild / scalaVersion       := scala213
+ThisBuild / crossScalaVersions := Seq("2.12.17", scala213, scala3)
 
-val versions: Map[String, String] = {
-  import org.snakeyaml.engine.v2.api.{ Load, LoadSettings }
-  import java.util.{ List => JList, Map => JMap }
-  import scala.jdk.CollectionConverters._
-  val doc  = new Load(LoadSettings.builder().build())
-    .loadFromReader(scala.io.Source.fromFile(".github/workflows/scala.yml").bufferedReader())
-  val yaml = doc.asInstanceOf[
-    JMap[String, JMap[String, JMap[String, JMap[String, JMap[String, JList[String]]]]]]
-  ]
-  val list = yaml.get("jobs").get("test").get("strategy").get("matrix").get("scala").asScala
-  list.map { v =>
-    val vs = v.split('.'); val init = vs.take(vs(0) match { case "2" => 2; case _ => 1 });
-    (init.mkString("."), v)
-  }.toMap
-}
+ThisBuild / tlBaseVersion := "2.5"
+
+ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec.temurin("11"))
+
+ThisBuild / githubWorkflowBuildMatrixExclusions += // TODO
+  MatrixExclude(Map("scala" -> scala3, "project" -> "rootJVM"))
 
-val scalaVer                = versions("2.13")
-val scala3Ver               = versions("3")
 val tzdbVersion             = "2019c"
```

Comment:
Perhaps we should do 2022b or newer to exercise the parser fix

Reply:
Agree. Follow-up PR? I also want to do a follow-up PR that fixes the tests, because right now it is not testing against the JDK at all. It will be a bit of work to fix all of that.

Reply:
Sounds good